test/unit/util/test_swagger_client.py

```diff
@@ -17,6 +17,18 @@
 from unittest.mock import mock_open
 
 
+import time
+from hca.dss import DSSClient
+
+
+class MockDSSClient(DSSClient):
```

Comment:
I don't think this should be called a "mock" DSS client, since it isn't actually mocking anything, just fixing an attribute value. It still works exactly like the normal DSSClient and makes live API calls.

Reply:
maybe rename to `ShortLivedDSSClient` or similar